.github/workflows/checkov.yaml

```diff
@@ -0,0 +1,30 @@
+name: Checkov
+on:
+  workflow_call:
+    secrets:
+      SLACK_WEBHOOK:
+        required: true
+jobs:
+  checkov-scan:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.9
+      - name: Test terraform with Checkov
+        id: checkov
+        uses: bridgecrewio/checkov-action@v12
+        with:
+          directory: .
+          framework: terraform
+      - name: Slack Notification failure
+        if: failure()
+        uses: rtCamp/action-slack-notify@v2
+        env:
+          SLACK_USERNAME: Boldlinksig
+          SLACK_MESSAGE: ":bell: Job failed."
```

Comment:
Repo name is required, if this runs on many repos most likely we will endup with many alerts and without visually identify the failed workflows.